crates/hc_bundle/src/packing.rs

```diff
@@ -0,0 +1,150 @@
+#![forbid(missing_docs)]
+
+//! Defines the CLI commands for packing/unpacking both DNA and hApp bundles
+
+use crate::error::{HcBundleError, HcBundleResult};
+use mr_bundle::{Bundle, Manifest};
+use std::path::Path;
+use std::path::PathBuf;
+
+/// Unpack a DNA bundle into a working directory, returning the directory path used.
+pub async fn unpack<M: Manifest>(
+    extension: &'static str,
+    bundle_path: &std::path::Path,
+    target_dir: Option<PathBuf>,
+    force: bool,
+) -> HcBundleResult<PathBuf> {
+    let bundle_path = ffs::canonicalize(bundle_path).await?;
```

Comment:
[Apparently we need to never use canonicalize](https://github.com/rust-lang/rust/issues/42869) - it just universally breaks windows.

Reply:
Maybe check out [https://crates.io/crates/dunce](https://crates.io/crates/dunce)?

Reply:
Nice find -- Maybe I will implement `ffs::canonicalize` to use `dunce::canonicalize` and we'll use that exclusively.

Reply:
Done